src/jdk.compiler/share/classes/com/sun/tools/javac/code/Lint.java

```diff
@@ -278,6 +278,12 @@ public enum LintCategory {
          */
         INCUBATING("incubating", false),
 
+        /**
+         * Warn about code in identity classes that wouldn't be allowed in early
+         * construction due to a this dependency.
+         */
+        INITIALIZATION("initialization"),
```

Comment:
IMHO this should contain the word `strict` somewhere.

Reply:
sorry, do you mean the category or the comment?